newIDE/app/src/MainFrame/Onboarding/OnboardingDialog.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+import { Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import userflow from 'userflow.js';
+import Dialog from '../../UI/Dialog';
+import FlatButton from '../../UI/FlatButton';
+import optionalRequire from '../../Utils/OptionalRequire';
+import { getProgramOpeningCount } from '../../Utils/Analytics/LocalStats';
+import { isMobile } from '../../Utils/Platform';
+import Window from '../../Utils/Window';
+import { loadPreferencesFromLocalStorage } from '../Preferences/PreferencesProvider';
+import { getUserUUID } from '../../Utils/Analytics/UserUUID';
+import RaisedButton from '../../UI/RaisedButton';
+import { Line } from '../../UI/Grid';
+import { MarkdownText } from '../../UI/MarkdownText';
+import { showErrorBox } from '../../UI/Messages/MessageBox';
+import { ColumnStackLayout } from '../../UI/Layout';
+
+const electron = optionalRequire('electron');
+const isDev = Window.isDev();
+export let isUserflowRunning = false;
+
+const styles = {
+  imgContainer: {
+    marginBottom: 16,
+  },
+};
+
+const onboardingText = `
+In 5 minutes, you will have:
+  - Created a game
+  - Learned the fundamentals of GDevelop
+
+(🇬🇧 The tour is only available in English)  
+We highly recommend it!
+`;
+
+const OnboardingDialog = () => {
+  const [open, setOpen] = React.useState(false);
+  const [loading, setLoading] = React.useState(false);
+
+  const startUserflow = React.useCallback(async () => {
+    try {
+      setLoading(true);
+      if (isDev) {
+        userflow.init('ct_y5qogyfo6zbahjejcbo3dybnta');
+      } else {
+        userflow.init('ct_paaz6o2t2bhlrlyi7a3toojn7e');
+      }
+      userflow.on(
+        // Undocumented legacy userflow event that is fired
+        // "when a flow either becomes active or removed"
+        // (tip given by a tech member of Userflow - it shouldn't be removed
+        // in the near future given the fact that some of their users still use it).
+        'flowvisibilitychange',
+        isRunning => {
+          isUserflowRunning = isRunning;
+        }
+      );
```

Comment:
Still consider putting this into a "initialiseUserFlow" method that will automatically check with a boolean `isUserflowInitialised` (set to false originally) if the init must be done again, and if not, it can be skipped.

This will make this startUserflow method simpler to read, and later if we start other flows we can just call this method :)